### PR TITLE
build01: add ssh key to masters

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @openshift/test-platform

### CODE REVIEW

**Comment by hongkailiu** [comment]
/hold

https://coreos.slack.com/archives/C02BPRL989H/p1629387065062200

seems not working without a kubelet.

File: clusters/build-clusters/01_cluster/assets/99-master-ssh_machineconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-master-ssh
+spec:
```
Comment (alvaroaleman) [inline clusters/build-clusters/01_cluster/assets/99-master-ssh_machineconfig.yaml:7]: Can we remove all the empty sections to be sure this wont reset any config?

Comment (hongkailiu) [reply clusters/build-clusters/01_cluster/assets/99-master-ssh_machineconfig.yaml:7]: I will try to apply it after the incident. I guess no need to create the noise right now.

Comment (hongkailiu) [reply clusters/build-clusters/01_cluster/assets/99-master-ssh_machineconfig.yaml:7]: Updated.

**Review by alvaroaleman — APPROVED** [review]

**Review by alvaroaleman — COMMENTED** [review]
/hold cancel

**Pull request merged**